liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -206,41 +210,70 @@ public void stopServer() {
 
         @Override
         public void startServer() {
-            try {
-                if (serverTask == null) {
-                    serverTask = initializeJava();
-                }
-                copyConfigFiles();
-                serverTask.setClean(clean);
-                serverTask.setOperation("start");
-                // Set server start timeout
-                if (serverStartTimeout < 0) {
-                    serverStartTimeout = 30;
-                }
-                serverTask.setTimeout(Long.toString(serverStartTimeout * 1000));
-                serverTask.execute();
-
-                if (verifyTimeout < 0) {
+        	final Semaphore serverTaskInitialized = new Semaphore(0);
+        	
+            Thread serverThread = new Thread(new Runnable() {
+
+				@Override
+				public void run() {
+					try {
+		                if (serverTask == null) {
+		                    serverTask = initializeJava();
+		                }
+		                
+		                messagesLogFile = new File(serverTask.getOutputDir() + "/" + serverTask.getServerName() + "/logs/messages.log");
+		                
+		                if (messagesLogFile.exists()) {
+		            		// Delete previous log file so it doesn't interfere with
+		            		// the waitForStringInLog method.
+		            		messagesLogFile.delete();
```

Comment:
We shouldn't delete any logs.  In what scenario would the messages log already exist?  It is usually a new file every time the server starts up.